app/styles/_global.scss

```diff
@@ -3,3 +3,14 @@
 #toast-container > div { // stylelint-disable-line selector-max-id
     opacity: 1;
 }
+
+// microfeedback-button styling
```

Comment:
The auto-generated class is available as `styleNamespace` on the component -- if you apply that to the alert you could move these styles to `components/feedback-button/styles.scss`

Reply:
@aaxelb The markup is generated by microfeedback-button, not by ember, so we can't use the auto-generated classes.

Reply:
That said, I suppose I could add an option to `microfeedback-button` so that you could add a custom class. Something like

```js
microfeedback({
  className: 'OSFMicroFeedbackButton',
})
```

```scss
& .OSFMicroFeedbackButton {
   ...
}
```

But that can be a future improvement.

I think the real fix would be to set the correct base font size on the `html` element. For some reason, ember-bootstrap (I think?) seems to be setting `font-size` to `10px` on `html` then overrides it to `14px` on `body`. Since microfeedback uses `rem` for font sizes, the text ends up being too small, hence these overrides.

If we could instead set `font-size` to `100%` on `html`, the CSS in this PR would be unnecessary. That might have unintended consequences, so I decided not to do that in this PR.

Reply:
Heh, I was already submitting a [feature request](https://github.com/microfeedback/microfeedback-button/issues/26). You would still have to use `styleNamespace`, though:
```js
microfeedback({
  className: this.get('styleNamespace'), // or maybe `${this.get('styleNamespace')}__microfeedback`
})
```

We should definitely set font size correctly, and probably use `rem` throughout. But that might fall under the [responsive font size analysis](https://openscience.atlassian.net/browse/EMB-91). I'm fine with these global styles for now.